doc/docs.md

```diff
@@ -5339,8 +5339,8 @@ module abc
 pub struct Xyz {
 pub mut:
 	a int
+	// the tags will produce a notice, since the deprecation date is in the far future
 	d int [deprecated: 'use Xyz.a instead'; deprecated_after: '2999-03-01']
-	// the tags above, will produce a notice, since the deprecation date is in the far future
```

Comment:
This comment starts with `the tags above`, it should stay below the field to which it refers. If vfmt moves it above, that is a problem :-| .

Reply:
Ok, I see, I'll fix it.